### ISSUE TITLE
friend save

### ISSUE THREAD

**Issue by xzcoli** [RusJJ/GorillaFriends#7]
basiclly when i add someone, leave and rejoin the lobby he is not in friend anymore i even tried to relaunch my game and reinstall the mod and it still does that

**Comment by xzcoli** [comment]
after doing that you can't add friends anymore ¨in forest only¨

**Comment by xzcoli** [comment]
if you want a demonstration add me ._kid._. I will screen share with you

**Comment by RusJJ** [comment]
Please check experimental fix if possible: https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1

**Comment by ghost** [comment]
> Please check experimental fix if possible: https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1

Friends still don't save and recently played with doesn't work

**Comment by xzcoli** [comment]
> Please check experimental fix if possible: https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1

i will

**Comment by xzcoli** [comment]
> Please check experimental fix if possible: https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1

yea it does not save on this one neither

**Comment by RusJJ** [comment]
Appreciate if you will try this: https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1-v2

Dont have much time rn to find a fix

**Comment by xzcoli** [comment]
sorry for late respond, but i will try it and test every single issue i could think of :)

**Comment by xzcoli** [comment]
> Appreciate if you will try this: https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1-v2
> 
> Dont have much time rn to find a fix

idk is its because i have a shity pc but when ever i join a lobby it just start freezing every 2 seconds

**Comment by RusJJ** [comment]
Probably completely working version, cant really check rn (but i did earlier):

https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1-v4

**Comment by ghost** [comment]
> Probably completely working version, cant really check rn (but i did earlier):
> 
> https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1-v4

Yep works great no issues friends save Scoreboard is normal everything works

**Comment by xzcoli** [comment]
> Probably completely working version, cant really check rn (but i did earlier):
> 
> https://github.com/RusJJ/GorillaFriends/releases/tag/pre-1.2.1-v4

great, but like the last version I freeze every 2 second but I don't blame you or the code, I blame my pc
